definitions/ext-dell_compellent/definition.yml

```diff
@@ -0,0 +1,39 @@
+domain: EXT
+type: DELL_COMPELLENT
+
+synthesis:
+  name: clusterCI
+  identifier: clusterCI
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: eventType
+    prefix: compellent_
+  - attribute: clusterCI
```

Comment:
We already check that the attribute for the name or the indetifier exists in the data points so this condition is not needed

Reply:
removed second condition from all definitions